docusaurus/docusaurus.config.js

```diff
@@ -0,0 +1,78 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title:                 'Rancher DevKit',
+  tagline:               'Rancher development kit',
+  url:                   'https://bisht-richa.github.io',
+  baseUrl:               '/dashboard/',
```

Comment:
Why are we going to use `/dashboard/`?
Is it planned to have more docs?

Reply:
As clarified (thanks @bisht-richa ), in GH pages the path is the type of `https://<USERNAME>.github.io/<PROJECT_NAME>`, so this would be the documentation for mentioned path.